ControlRoomApplication/ControlRoomApplication/Controllers/Communications/RemoteListener.cs

```diff
@@ -70,150 +70,154 @@ public void TCPMonitoringRoutine()
                     waitingForConn = true;
                     new Thread(() =>
                     {
-                        TcpClient client = server.AcceptTcpClient();
-                        logger.Debug(Utilities.GetTimeStamp() + ": TCP Client connected!");
-
-                        data = null;
-                        // Get a stream object for reading and writing
-                        NetworkStream stream = client.GetStream();
-
-                        int i;
-
-                        // Loop to receive all the data sent by the client.
-                        if((i = readFromStream(stream, bytes)) != 0 )
+                        using (TcpClient client = server.AcceptTcpClient())
```

Comment:
I see you found out one way to use the `using` statement and that's good! There is a simpler way though where you can just define it in the variable declaration like in the second picture [here](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement). This way, we don't need to indent as much and can have nicer looking code.

- [ ] Simplify `using` statement